src/Yaapii.Xambly/Cursor/DomCursor.cs

```diff
@@ -57,7 +57,7 @@ public IEnumerator<XNode> GetEnumerator()
         /// <returns>Enumerator</returns>
         IEnumerator IEnumerable.GetEnumerator()
         {
-            return GetEnumerator();
+            return this.GetEnumerator();
```

Comment:
It is unnecessary to put "this" before class methods because there can be no other methods than class methods.
It only makes readability worse.